---
title       : Dermatology Branch Journal Club
subtitle    : 2014-01-31
author      : Kenneth Daily
url         : {lib: libraries}
job         : 
framework   : shower        # {io2012, html5slides, shower, dzslides, ...}
highlighter : highlight.js  # {highlight.js, prettify, highlight}
hitheme     : tomorrow      # 
widgets     : [mathjax, bootstrap, quiz]
mode        : selfcontained # {standalone, draft}
--- .cover #Cover

```{r setup,echo=FALSE, warning=FALSE, message=FALSE}
library(ggplot2)
library(xtable)
library(plyr)

options(xtable.type="html", xtable.caption.placement="top")

opts_chunk$set(comment=NA, echo=FALSE, warning=FALSE, message=FALSE)
```

## Dermatology Branch Journal Club<br/>Kenneth Daily<br/>2014-01-31

<img src="./assets/img/cover.png" width="1024" alt="circos">

---

## DNA tumor viruses

---

## What is HPV

---

## HPV infection

---

## HPV integration

---

## Can HPV integration be detected by whole genome sequencing?

---

## Fig 1A

The HPV view of integration events

```{r}
## clonal relationship b/c of unique homozygous viral mutations
## Variable coverage indicates breakpoints/rearrangements
## Unique breakpoints, rearrangements
## E6/E7 oncogenes retained
## RNASeq positive for HPV viral genes (supp fig 2)
```

---

## Fig 1B - breakpoints detected in HPV genome

```{r}
## Unappreciated view of intraviral rearrangements
```

---

## Supp Fig 1a
---

## Supp Fig 1b
---

## Supp Fig 1c

---

## Supp Fig 1i-j

---

## Variation in the host genome (Supp Fig 3)

Comparison of mutational spectra in HPV-positive vs. HPV-negative cancer samples.

---

## Focal, non-recurrent HPV integration clusters (Figure 2)

HPV integrants cluster in focal regions within samples, but uniquely throughout the genome between samples

```{r}
## specific, chromosomal regions in different cell lines
## confirmed with FISH and SKY
## Confirmed most except low copy number
## HPV FISH signals found in identical cytogenetic regions of duplicate chromosomal copies, but with variable intensities
## suggests integration preceded chrom duplic and subsequent divergence in viral copy number. 
## 6/9 
```

---

## HPV insertions flank CNVs

Is there a relationship between HPV integration and host genome CNVs?

```{r}
## breakpoints less numerous than viral CN
## Hypothesized due to amplification of viral integrants and flanking genomic sequence -> redundant identical BPs
## observe focal genomic CNVs in 8/9 HPV-positive cases
## found at amplfications ranging from 1.5x (HMS001) to 58x (UPCI:SCC90) increase
## also occurred near deletions from 487bp (HMS001) to 234kb (UPCI:SC0090)
## All confirmed by qPCR, PCR, Sanger seq
```

---

##

What is the detailed structure of the focal rearrangements involving HPV integration?

---

##

```{r}
## Used the breakpoints and CNVs identified through WGS to build connectivity maps
## Confirmed with targeted PCR, Sanger sequencing and chromosomal walking
```

---

## SiHa cells (Figure 4A)

```{r}
## Two insertional breakpoints flank a 2x amplified 300kb segment with an internal 72kb deletion
## ends up with a concatemerized head-to-tail genomic/HPV segments (two HPV integrants bounded by identical breakpoints)
```

---

## HMS001 cells (Figure 4B)

```{r}
## two HPV insertional breakpoints spannind a small deletion
## Adjacent host genome frgas and HPV seqs duplicated in tandem - results in two identical HPV integrants with an intrachromosomal rearrangement
```

--- 

## More complex integration + CNV rearrangements

Figure 5

```{r}
## resolved complex linear rearrangements in all but one HPV-positive cell lines
```
