Derm Branch Journal Club 2014-01-31 
===================================

Presentation of "Genome-wide analysis of HPV integration in human cancers reveals recurrent, focal genomic instability".

Introduction
----------

## HPV background
* HPVs cause 610K cancers
* main cause of cervical cancer
* E6 and E7 oncoproteins are transforming (inactivate TSGs P53 and RB1)
* early in infection, HPV replicates episomally
* Integration provides selective growth advantage, increased genomic instability
* increase in E6/E7 expression due to loss of E2
* associated with structural abnormalities

## HPV integration
* HPV infects keratinocytes in basal layer of epithelium
* Infected keratinocytes do not undergo terminal differentiation
* Virus replicates as episomal elements
* estimated 10% of cases a transforming type of HPV infection evolves. 
* characterized by the deregulation of viral oncogenes E6 and E7 in cycling cells 
* results in chromosomal instability and the accumulation of mutations

* Integration into host genome increases with severity
* Oncoproteins
* E6 up, p53 down (downregulates apoptosis pathway genes)
* E7 up, RB1 down (disrupts G1/S phase transition of cell cycle)


Results
----------

## Table 1
* Looked at HPV integration in 10 cell lines, two tumor samples

## Variability in HPV genomes and transcriptomes (Fig. 1A)
* 2x100bp PE whole genome sequencing
* signif reads aligned to HPV in V+ samples

* clonal relationship b/c of unique homozygous viral mutations

* Variable coverage indicates breakpoints/rearrangements
* Total of 111 unique insertional breakpoints; 105 confirmed by PCR
* Some flanked missing viral segments (indicating loss at integration; ex. UM-SCC-47)
* Unique breakpoints, rearrangements
* E6/E7 oncogenes retained (confirmed by RNAseq)
* RNASeq positive for HPV viral genes (supp fig 2)

* Tumor B seemed to have constant read depth - indication of presence of episomal HPV

## HPV and the cellular genome
* Inverse relationship b/t HPV integration, TP53, CDKN2A inactivating mutations
* Also detected mutations/aberrations in other HNSCC gees (PTEN, PIK3CA, NOTCH, TP63)
* Also general recurrent levels of previously reported CNV (5p, gain, 3p loss)

## Focal clustering of HPV integrants

* HPV integrants cluster in focal regions within samples, but uniquely throughout the genome between samples
* specific, chromosomal regions in different cell lines
* confirmed with FISH and SKY
* Confirmed most except low copy number
* HPV FISH signals found in identical cytogenetic regions of duplicate chromosomal copies, but with variable intensities
* suggests integration preceded chrom duplic and subsequent divergence in viral copy number. 
* 6/9 

## HPV insertions flank CNVs

* statistically significant association with CNVs
* observe focal genomic CNVs in 8/9 HPV-positive cases
* found at amplfications ranging from 1.5x (HMS001) to 58x (UPCI:SCC90) increase
* also occurred near deletions from 487bp (HMS001) to 234kb (UPCI:SC0090)
* All confirmed by qPCR, PCR, Sanger seq
* many examples where genes may be disrupted

## Resolving focal genomic amplifications and rearrangements

Using the breakpoints and CNVs identified through WGS, build connectivity maps.

* Confirmed with targeted PCR, Sanger sequencing and chromosomal walking

### SiHa cell line
* Two insertional breakpoints flank a 2x amplified 300kb segment with an internal 72kb deletion
* ends up with a concatemerized head-to-tail genomic/HPV segments (two HPV integrants bounded by identical breakpoints)

### HMS001 cell line
** two HPV insertional breakpoints spannind a small deletion
** Adjacent host genome frags and HPV seqs duplicated in tandem - results in two identical HPV integrants with an intrachromosomal rearrangement

### Other cell lines
* resolved complex linear rearrangements in all but one HPV-positive cell lines

### Tumor A

2MB region on chrom 5 - very complex, data suggests multiple looping structures forming, involved a candidate oncogene, PDE4D

### Tumor B

Partial fragment of HPV integrated in an intrachromosomal translocation

## Looping model of Your Favorite Gene (Fig. 4C)

* HPV integrants bridge non-contiguous genomic sequence by connecting nicked sites
* Then, the replication of a transient circularized genomic segment would result in focal amp of breakpoints involving HPV and flanking host genomic sequences
* Then, distal free ends of amplified strands could recombine with non-contiguous sequences and undergo DNA repair (ATR and ATM repair pathways are used in viral replication)
* This would result in concatemerized linear structures bounded by recurrent, identical breakpoints

RC replication has been observed in HPV (extracts from human epithelial cells induce rolling circle replication of the HPV16 origin-containing plasmid in vitro - http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2443.2010.01458.x/full)

## HPV integrants disrupt cellular genes by multiple mechanisms

### DIAPH2 in 
## deletion and rearrangement of DIAPH2 locus, which plasy a role in sister-chromatid separation
## resulted in alt spiced viral host fusion genes
## viralhost fusion transcripts of DIAPH2 were expressed while the native transcripts was not
Discussion
----------

Issues
------

* Use of cell lines
* Difficulty in sequencing repetitive regions
* No mention of reports of HPV integration mechanism possibilities
* No mention of 'tandem repeats', or head-to-tail concatemers, of HPV, which have previously been reported, including in the cell line CaSki, which is reported to have 500 tandem repeats of HPV16!

Other than to mention that ATR or ATM pathways may play a role, since they are used in viral replication, and their looping structures could be a by-product of aberrant replication of the HPV integrant. But this is initiated by E2 - but it is frequently disrupted?

* rolling circle amp is probably not good for HPV; herpes viruses use it but have viral encoded cleavage protein, which papillomaviruses lack
* Some suggestion that DNA damage to HPV could lead to accidental RCA followed by integration via host DS-break repair mechanisms

Other DNA viruses (MCV)
--------------------

* We see similar signatures in exome sequencing data
